src/analysis/retail/shaman/enhancement/modules/Abilities.tsx

```diff
@@ -414,12 +47,9 @@ class Abilities extends CoreAbilities {
       },
       {
         // This is no error. We actually use the elemental shaman elemental blast spell id.
-        spell: TALENTS_SHAMAN.ELEMENTAL_BLAST_ELEMENTAL_TALENT.id,
-        enabled: combatant.hasTalent(TALENTS_SHAMAN.ELEMENTAL_BLAST_ENHANCEMENT_TALENT),
-        charges: combatant.getMultipleTalentRanks(
-          TALENTS_SHAMAN.ELEMENTAL_BLAST_ENHANCEMENT_TALENT,
-          TALENTS_SHAMAN.LAVA_BURST_TALENT,
-        ),
+        spell: TALENTS.ELEMENTAL_BLAST_ELEMENTAL_TALENT.id,
+        enabled: combatant.hasTalent(TALENTS.ELEMENTAL_BLAST_ENHANCEMENT_TALENT),
+        charges: 1, //combatant.getMultipleTalentRanks(TALENTS.ELEMENTAL_BLAST_ENHANCEMENT_TALENT, TALENTS.LAVA_BURST_TALENT),
```

Comment:
stray comment?

Reply:
Yep, was also related to the other charge change. Reverted.